strider/trapi.py

```diff
@@ -445,21 +445,25 @@ async def fill_categories_predicates(
     for node in qg['nodes'].values():
         node_id = node.get('ids', None)
         if not node_id:
-            continue
-        if not isinstance(node_id, list):
-            node_id = [node_id]
+            if 'biolink:Gene' in node['categories'] and 'biolink:Protein' not in node['categories']:
```

Comment:
A good practice is to touch as little code as possible so that a) PRs are easy, and b) git blames are helpful. Here we can just add these lines before the `continue` and not touch the rest.

Reply:
I guess that's true. I'd always been told that continue statements were sort of "bad practice", so since the continue statement was no longer necessary, now that there was something actually in the if-block, I though it would be good to remove it.